modules/liberty/src/main/java/org/testcontainers/containers/liberty/LibertyAdapter.java

```diff
@@ -112,4 +112,8 @@ public ImageFromDockerfile getDefaultImage(File appFile) {
         return image;
     }
 
+    @Override
+    public String getReadinessPath() {
+        return "health/ready";
```

Comment:
lets have the Liberty adapter return null for now, because `health/ready` will only be available if the user has enabled the `mpHealth-2.0` feature in their Liberty config file